### ISSUE TITLE
Add Dropbox support as a datastore

### ISSUE THREAD

**Issue by toroidal-code** [spectra-music/spectra#4]
Add dropbox support for the location of the files to play

This will probably involve either adding 
1. another abstraction to the filesystem in the models
OR
2. A directive in the track model to indicate whether the track is local or remote

**Comment by nickserv** [comment]
:+1: Awesome idea. I'm assuming this means that audio files in your Dropbox could be streamed instead of downloading your music folder(s) completely?

**Comment by toroidal-code** [comment]
Yup, exactly. Music would remain on dropbox, and spectra could handle organization and playing of the files.

**Comment by nickserv** [comment]
https://www.dropbox.com/developers/core/start/ruby

**Comment by zellio** [comment]
@nicolasmccurdy - Yeah that's the goal. There is a chance that this get's done with more javascript than ruby. But I have to look at the ruby api before I say that.
